applications/RANSModellingApplication/tests/ChannelFlowTest/channel_flow_step_k_epsilon_transient_drag_mpi_ref.dat

```diff
@@ -0,0 +1,9 @@
+# Body fitted drag for model part FluidModelPart.Slip2D
```

Comment:
why do you need a separate results file for MPI?
IMO the results should be the same in OMP and MPI

Reply:
there is no omp version of this test. It is only MPI :)